seqfold/dna.go

```diff
@@ -0,0 +1,519 @@
+package seqfold
+
+var DNA_COMPLEMENT = Comp{'A': 'T', 'T': 'A', 'G': 'C', 'C': 'G', 'N': 'N'}
```

Comment:
Can use the complement tools in `transform` for this.

Reply:
Yes, but it seems to work on only for DNA, what about RNA?

Also is `unsafe` really necessary? But that's another issue altogether.